.changeset/polite-bags-act.md

```diff
@@ -0,0 +1,17 @@
+---
+"@salt-ds/lab": minor
+---
+
+- Changed `StepperInput`'s `onChange` to return a value and an optional event.
+- Replaced functionality of Page up and Page down keys in `StepperInput` with Shift + Up arrow and Shift + Down arrow. These keyboard controls change the value by the `block` amount.
+- The `block` amount now explicitly defines the value that is increment or decrement, not a multiplier of `step`.
```

Comment:
There was conflicting docs in uitk about this, and I think this way gives more control as the block doesn't need to be an exact multiple of the step.

Reply:
I believe the intent was more around a best practice. If we're removing it from the jsdoc can you make sure it's on the site docs cc @navkaur76

Reply:
yeah I can update in the docs PR